classes/phing/Phing.php

```diff
@@ -1048,6 +1055,38 @@ public static function printVersion()
     }
 
     /**
+     * Creates generic buildfile
+     */
+    public static function init()
+    {
+        $homeDir = self::getProperty(self::PHING_HOME);
```

Comment:
I think we could take the app start directory here `self::getProperty('application.startdir')`